articles/rules/redirect.md

```diff
@@ -32,18 +31,22 @@ function (user, context, callback) {
 
 Once all rules have finished executing, the user will be redirected to the specified URL.
 
-## What to do after redirecting
+## What to do afterwards
 
 An authentication transaction that has been interrupted by setting `context.redirect` can be resumed by redirecting the user to the following URL:
 
 ```text
 https://${account.namespace}/continue?state=THE_ORIGINAL_STATE
 ```
 
+::: panel-info State
+State is an opaque value the clients adds to the initial request that Auth0 includes when redirecting back to the client. We highly recommend using this param, since it can be used by the client to prevent [CSRF attacks](/security/common-threats#cross-site-request-forgery-xsrf-or-csrf-). By `THE_ORIGINAL_STATE` we mean the value you provided during user authentication.
+:::
+
 When a user has been redirected to the `/continue` endpoint, all rules will be run again.
 
-::: panel-danger Caution:
-Make sure to send back the original state to the `/continue` endpoint, otherwise Auth0 will lose the context of the login transaction.
+::: panel-danger Caution
+Make sure to send back the original state to the `/continue` endpoint, otherwise Auth0 will loose the context of the login transaction.
```

Comment:
`lose` is correct in this situation. Also, I think it might be nice to elaborate on what "losing the context of login transaction" means. Something like:

```
Make sure to send back the original state to the `/continue` endpoint, otherwise Auth0 will lose the context of the login transaction and the user will not be able to login due to to an `invalid_request`.
```